Utilities.java

```diff
@@ -1,55 +1,56 @@
 package com.utils;
 
 public class Utilities {
-  private static final int NUM = 114;
+  private static final int NUM_FILES = 114;
 
-private static void findJavaFiles(File parentDirectory, List<File> files) {
-  for (File file : parentDirectory.listFiles()) {
+  private static void findJavaFiles(File parentDirectory, List<File> files) {
```

Comment:
Esto no, por que no hace claridad con la funcion
```suggestion
private static void findJavaFiles(File parentDirectory, List<File> files) {
```